### ISSUE TITLE
Receipts doesn't work anymore

### ISSUE THREAD

**Issue by Sevenlive** [bluewalk/lidlplus-php-client#4]
They Receipts doesn't work, probrably because of a new API-Endpoint. I found out, that its now accessable though 

https://appgateway.lidlplus.com/tickets/api/v1/DE/list/1?

instead of 

https://appgateway.lidlplus.com/app/v19/DE/tickets/list/1

Also the Overall API-Version could be bumped up to v24 instead of v19.

If you want, i could add a PR.

**Comment by bluewalk** [comment]
Hi, seems like the whole endpoint has changed, need to spend some time figuring out which endpoint is used now for the ticket details as well.

**Comment by Sevenlive** [comment]
Yes they did. Took me a while to find that out as well. My Way included a Rooted Emulator and Frida to brick the certificate pinning.

**Comment by bluewalk** [comment]
I've been trying to hack the APK, got the certificate pinning disabled but my new signature broke google maps so couldn't choose a store and got stuck in the onboarding process.

**Comment by Sevenlive** [comment]
I used this guide, that was pretty helpful.

[https://httptoolkit.tech/blog/frida-certificate-pinning/](https://httptoolkit.tech/blog/frida-certificate-pinning/)

That worked for me. I also tried stuff like apk-mitm before, but the whole apps crashed with that patched version.

**Comment by bluewalk** [comment]
You've done a great job at finding the new endpoints :D I have merged your pull request and will take care of the packagist submission.
